float64 value
---
float64[] next